share/revng/test/tests/abi/prepare-functions-binary.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
```

Comment:
1. Always use `#!/usr/bin/env bash`
2. Drop the shebang and `set -euo pipefail`, those belong only to the actual executable script, not on things that are supposed to be `source`'d.

Reply:
1. I just thoughtlessly copied it from the old script 😅.
2. Done.

3. On that note, is there an easy way to prevent running it? Something akin to python's
```python
if __name__ == "__main__":
  do_stuff()
else
  error_out()
```
?

Reply:
3. Not that I know of.

Reply:
Gotcha, sad. I guess this will have to do then.